# This message expressed the rotation and translational vector

geometry_msgs/Vector3 rotational
geometry_msgs/Vector3 translational
